tasks/task_save.c

```diff
@@ -575,35 +593,124 @@ static void task_save_handler_finished(retro_task_t *task,
    free(state);
 }
 
-static void *get_serialized_data(const char *path, size_t serial_size)
+static size_t content_align_size(size_t size)
+{
+   /* align to 8-byte boundary */
+   return ((size + 7) & ~7);
+}
+
+static bool content_get_rastate_size(rastate_size_info_t* size)
+{
+   retro_ctx_size_info_t info;
+
+   core_serialize_size(&info);
+   if (!info.size)
+      return false;
+
+   size->coremem_size = info.size;
+   /* 8-byte identifier, 8-byte block header, content, 8-byte terminator */
+   size->total_size = 8 + 8 + content_align_size(info.size) + 8;
+
+#ifdef HAVE_CHEEVOS
+   size->cheevos_size = rcheevos_get_serialize_size();
+   if (size->cheevos_size > 0)
+      size->total_size += 8 + content_align_size(size->cheevos_size); /* 8-byte block header + content */
+#endif
+
+   return true;
+}
+
+size_t content_get_serialized_size()
```

Comment:
Use (void) in function declarations/prototypes instead of ().

Reply:
done